tests/jeos/glibc_locale.pm

```diff
@@ -208,4 +212,10 @@ sub run {
     reset_consoles;
 }
 
+sub post_fail_hook {
+    script_run 'locale';
```

Comment:
Could you add a comment explaining why those are needed?

Reply:
it is a locale test, so you wanna see the settings at the time of the failure.

Reply:
```suggestion
    # Print debugging information
    script_run 'locale';
```

Reply:
```suggestion
    # Print debugging information
    enter_cmd 'locale';
```